pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -66,5 +66,7 @@ extend type Mutation {
 extend type Query {
   mtoMilestone(id: UUID!): MTOMilestone!
   @hasAnyRole(roles: [MINT_USER, MINT_MAC]) #TODO (mto) is there any reason a MAC user shouldn't query this?
+  mtoMilestonesWithNoLinkedSolutions(modelPlanID: UUID!): [MTOMilestone!]!
```

Comment:
I suspect this should live on the mtoMatrix under model plan, instead of directly on the top level query.

Reply:
And, presumably, in that change we could drop the `modelPlanID` param, yeah?

Reply:
Yes, this is done locally and will be pushed up shortly. We can source from the mto matrix instead.